tests/network-tests/get-host-ip.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    # We could get the specific network address for the docker network with
+    #   docker network inspect --format='{{range .IPAM.Config}}{{.Gateway}}{{end}}' joystream_default
+    ip route | awk '/default/ {print $3}'
```

Comment:
For me locally this approach didn't work (`ip route | awk '/default/ {print $3}'`).
It returned `192.168.0.1`, while:
`docker network inspect --format='{{range .IPAM.Config}}{{.Gateway}}{{end}}' joystream_default`
Returned `172.19.0.1`

When set the endpoints to `172.19.0.1:3333` and `172.19.0.1:3334` everything worked as expected.
Should we use `docker network inspect --format='{{range .IPAM.Config}}{{.Gateway}}{{end}}' joystream_default` approach instead?

Reply:
The only downside of course is any client connecting to `172.19.0.1` would have to be from the same host. For the the purpose of local testing this is acceptable I think.

Reply:
done in 23ff187